src/az-databricks/README.md

```diff
@@ -1,46 +1,122 @@
-# VNet injection of Azure Databricks workspace
-
-<!-- Replace "Recipe Template" title with name of the recipe. -->
+# Azure Databricks Deployment in Secured Networking Environment
 
 ## Scenario
 
-<!-- Describe the usage scenario for this template.  Describe the challenges this recipe aims to address. -->
-This scenario aims to address the challenge of correctly configuring an Azure Databricks within a VNet including ensuring appropriate connectivity with common services such as Storage and Key Vault.
+This scenario aims to provide a sample architecture to deploy Azure Databricks in a secured networking environment using various available configuration options.
+
+## Problem Summary
+
+Azure Databricks provides a secure networking environment by default, but it also provides several additional features to further harden the network security. Here is a list of some of these features which the recipe uses.
+
+- [VNet Injection](https://learn.microsoft.com/azure/databricks/administration-guide/cloud-configurations/azure/vnet-inject)
+- [Secure Cluster Connectivity](https://learn.microsoft.com/azure/databricks/security/network/secure-cluster-connectivity)
+- [IP Access List](https://learn.microsoft.com/azure/databricks/security/network/ip-access-list-workspace)
+- [Azure Private Link](https://learn.microsoft.com/azure/databricks/administration-guide/cloud-configurations/azure/private-link)
+
+This recipe aims to provide developers a starting point with IaC(Infrastructure as Code) example of deploying Azure Databricks into a VNet and [enabling Azure Private Link as a standard deployment](https://learn.microsoft.com/azure/databricks/administration-guide/cloud-configurations/azure/private-link-standard). Please note that this is just one of the many possible configurations in which such an environment can be deployed based on your requirements.
+
+## Architecture
+
+![architecture](./media/databricks-vnet-architecture.drawio.svg)
```

Comment:
One minor suggestion: Could you highlight the correct access path via VNET for the user? (either by using a green tick or just making the routes in "bold")

As a first time reader, it wasn't obvious on the arch. It's only clear after reading the content blocks below.